django_airavata/apps/api/views.py

```diff
@@ -734,6 +737,147 @@ def get(self, request, format=None):
             data_product, context={'request': request})
         return Response(serializer.data)
 
+experiment_data_storage = FileSystemStorage(
```

Comment:
experiment_data_storage isn't needed? Also get_user_files and the other functions are duplicated here.